lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -466,17 +467,33 @@ void OpenCLFunction::executeConvolution(const OCLConvolutionInst *CC) {
   // VWN and VWM should be tunable.
   addStringOption(options, "VWM", "4");
   addStringOption(options, "VWN", "4");
+  if (isQuantized) {
+    addStringOption(options, "QUANTIZED", "");
```

Comment:
`QUANTIZED` was required when you had a version that had a single kernel with a bunch of "ifdefs" to handle quantized and non-quantized convolutions. Once you introduced two separate kernels, there is no need for all this macro-magic. Each kernel can have its own set of macro-definitions inside the kernels source file. No need to pass something from here.